src/client/sky.h

```diff
@@ -79,12 +197,50 @@ class Sky : public scene::ISceneNode
 		m_bgcolor = bgcolor;
 		m_skycolor = skycolor;
 	}
-	void setBodiesVisible(bool visible) { m_bodies_visible = visible; }
+	void updateSkyColors(video::SColor bg_day, video::SColor bg_dawn,
```

Comment:
why is this called `update` unlike the others who are called `set`?

Reply:
Perhaps I had a typo, fixed that for consistency.